registration/src/main/java/org/spine3/samples/lobby/registration/conference/ConferenceProjection.java

```diff
@@ -156,27 +160,54 @@ private void addOrRemoveSeats(int newQuantity, int oldQuantity, SeatTypeId id) {
         final int difference = newQuantity - oldQuantity;
         if (difference > 0) {
             sendAddSeatsRequest(id, difference);
-        } else if(difference < 0) {
+        } else if (difference < 0) {
             sendRemoveSeatsRequest(id, abs(difference));
         }
     }
 
     private void sendAddSeatsRequest(SeatTypeId seatTypeId, int quantity) {
         final AddSeats message = AddSeats.newBuilder()
-                .setConferenceId(getState().getId())
-                .setQuantity(newSeatQuantity(seatTypeId, quantity))
-                .build();
+                                         .setConferenceId(getState().getId())
+                                         .setQuantity(newSeatQuantity(seatTypeId, quantity))
+                                         .build();
         final Command command = create(message, newCommandContext());
-        commandBus.post(command);
+        commandBus.post(command, new StreamObserver<Response>() {
+            // TODO:12-09-16:dmytro.dashenkov: Handle completion or errors.
```

Comment:
Please either 
* address the `todo` items in this class before merging,
* or at least extract the empty observer creation into a static method.